js/src/behaviors/tabs.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tab controls
+ * @param callback - function of what the should happen on change
+ */
+
+export function initTabs(container: HTMLElement, callback = () => {}) {
+	initTabControlClickListeners(container, callback);
+}
+
+function initTabControlClickListeners(container: HTMLElement, callback: () => void) {
+	if (!container) {
+		return;
+	}
+
+	const buttons = Array.from(container.querySelectorAll('.tab-control')) as HTMLButtonElement[];
```

Comment:
need to use the window.addEventListener pattern instead. Can't assume tabs have all been rendered at page load.